.eslintrc

```
@@ -193,6 +198,35 @@
         "testing-library/no-dom-import": "error"
       }
     },
+    {
+      "files": [
+        "**/karma/**/*.js",
+        "**/test/**/*.karma.js",
+        "tests/js/**/*.karma.js"
```

I don't think this is applicable.

In `tests/js` we only have some config/setup files for Jest. If we'd ever have that for Karma, then they would probably be in `tests/karma`

```suggestion
        "tests/karma/**/*.js"
```

---

Definitely. Copy-paste...